libctru/include/3ds/applets/miiselector.h

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file miiselector.h
+ * @brief Mii Selector Applet (appletEd).
+ */
+
+#pragma once
+
+/// @cond INTERNAL
+#define _assert_struct_size(s, size) \
+	_Static_assert(sizeof(s) == (size), #s " is not of size " #size "!")
```

Comment:
This limits you to C11. I would do it like this:
```
/// @cond INTERNAL
#ifdef __cplusplus
#if __cplusplus >= 201103L
#define _assert_struct_size(s, size) \
	static_assert(sizeof(s) == (size), #s " is not of size " #size "!")
#endif
#else
#if __STDC_VERSION__ >= 201112L
#define _assert_struct_size(s, size) \
	_Static_assert(sizeof(s) == (size), #s " is not of size " #size "!")
#endif
#endif
/// @endcond

Reply:
Can we get rid of this assert_struct_size thing? I don't think it's necessary when it's already verified to be correct.

Reply:
Done.